Sources/CMakeLists.txt

```diff
@@ -2,3 +2,9 @@ add_subdirectory(ArgumentParser)
 if(BUILD_TESTING)
   add_subdirectory(ArgumentParserTestHelpers)
 endif()
+
+set_property(GLOBAL APPEND PROPERTY ARGUMENTPARSER_EXPORTS ArgumentParser)
```

Comment:
Could use ArgumentParser_EXPORTS but, that’s a purely style thing, it has 0 bearing on anything.

Reply:
Good to know, I'm not that familiar with CMake style conventions. I updated it since I was already making changes